util/src/servers/configuration/manufacturing_server.rs

```diff
@@ -59,13 +59,15 @@ pub struct DiunSettings {
 
 #[derive(Debug, Serialize, Deserialize, Clone, Copy)]
 pub enum MfgStringTypeString {
-    SerialNumber,
+    SerialNumber = 0,
+    SerialNumberExt = 1,
```

Comment:
There is no need to assign a constant to the enum, if we need to print them then it makes more sense to implement the [std::fmt::Display](https://doc.rust-lang.org/std/fmt/trait.Display.html#) trait

Reply:
agreed there is no specific need for assigned values (had added it for testing so removed it) , thanks.